data/tab-category.json

```diff
@@ -62,6 +62,20 @@
     "link": null
 },
 {
+    "tab": "starters",
+    "category": "rust-cookbook",
+    "labels": ["easy"],
+    "milestone": null,
+    "link": null
+},
+{
+    "tab": "impl",
+    "category": "rust-cookbook",
+    "labels": ["example", "tracking issue"],
```

Comment:
>  "labels": ["example", "tracking issue"],

Thanks @nrc  unfortunately I cannot see cookbook entries in the "impl" tab. I'm guessing that the issue labels need to match **all** listed "labels" instead of **any** (what I've thought). Am I correct?

Reply:
Yes, I believe so

Reply:
@nrc  Darn. I'll switch to milestone then https://github.com/nrc/find-work/pull/35